### ISSUE TITLE
Pop up Update or Update the app in App

### ISSUE THREAD

**Issue by MrConqueror** [mostafaalagamy/Metrolist#41]
### Checklist

- [X] I've checked that there is no other issue about this feature request.
- [X] This issue contains only one feature request.
- [X] The title of this issue accurately describes the feature request.

### Feature description

When there's a new Ver of the app i see here in github i have to uninstall the app so i can install the newer version because if i dont uninstal the old ver and install the new one in github it says App not installed as package appears to
be invalid.
![Screenshot_2024-08-19-18-28-02-822_com google android packageinstaller-edit](https://github.com/user-attachments/assets/380f30b5-6c9d-44a2-9754-822d2b03faff)

### Why do you want this feature?

I suggest you add a pop up Update in App or something 
It's hassle if there's new ver and i uninstall the old ver it erase all the download songs, All Gone.

**Comment by mostafaalagamy** [comment]
Does this always happen?

I believe it has been happening on the release of version 7.7.7 because I changed the signature. I think version 8 will be installed normally if you are on version 7.7.7.

![Screenshot_2024-08-19-14-09-17-294_com.github.android-edit.jpg](https://github.com/user-attachments/assets/26d32dc2-ea36-439c-8f66-2457622e919e)

I added an update checker that will show a notification in the settings when a new update is available, and I might consider adding a floating window as well.

![-w7lv9.jpg](https://github.com/user-attachments/assets/aa48558c-278f-4868-b616-208a5a4d2b96)

![Screenshot_2024-08-18-06-49-32-037_com.metrolist.musid-edit.jpg](https://github.com/user-attachments/assets/8338fe5e-311f-4057-9527-c64ca07fc2e6)

**Comment by MrConqueror** [comment]
Does this always happen? Idk dude wdym? 
 
I didn't know there was a new ver until i open github and saw your app we didn't receive a notification or pop up that says new update available or something

**Comment by mostafaalagamy** [comment]
> we didn't receive a notification or pop up that says new update available

Yes, I added it in the latest version 8.0.0, you will get in-app updates in the future starting from the next update.
